web/react/components/post_body.jsx

```diff
@@ -126,7 +126,7 @@ module.exports = React.createClass({
                 } else if (i < Constants.MAX_DISPLAY_FILES) {
                     postFiles.push(
                         <div className="post-image__column custom-file" key={fileInfo.name+i}>
-                            <a href={fileInfo.path+"."+fileInfo.ext} download={fileInfo.name+"."+fileInfo.ext}>
+                            <a href={fileInfo.path + (fileInfo.ext ? "." + fileInfo.ext : "")} download={fileInfo.name + (fileInfo.ext ? "." + fileInfo.ext : "")}>
```

Comment:
FYI The ternary operator will be outlawed by the style guide. You don't have to fix it here though. (you can if you want)